implementation/jwt-build/src/test/java/io/smallrye/jwt/build/JwtSignTest.java

```diff
@@ -573,6 +575,31 @@ public void testSignClaimsSymmetricKey() throws Exception {
         Assert.assertEquals("custom-value", claims.getClaimValue("customClaim"));
     }
 
+    @Test
+    public void testSignWithKeyStore() throws Exception {
+        JwtBuildConfigSource configSource = getConfigSource();
+        configSource.setUseKeyStore(true);
+        configSource.setSigningKeyLocation("/keystore.jks");
+        try {
+            String jwt = Jwt.claims()
+                    .claim("customClaim", "custom-value")
+                    .sign();
+
+            KeyStore keyStore = KeyUtils.loadKeyStore("keystore.jks", "password", Optional.empty(), Optional.empty());
```

Comment:
Might be good to make this new keystore a PKCS12 file to hit the alternate code branch.

Reply:
@MikeEdgar I've changed it to use a `PKCS12` keystore type, which can be confirmed with `keytool -list -storetype PKCS12 -keystore keystore.p12`, but at the test level I could not cause a failure to prove it is a PKCS12 store by reading it as JKS because by default it just works, 
```
KeyUtils.loadKeyStore("keystore.p12", "password", Optional.of("JKS"), Optional.empty());
```

due to the compatibility mode, and setting a system property is needed to disable it, https://www.oracle.com/java/technologies/javase/8u60-relnotes.html...